batch/batch/templates/batches.html

```diff
@@ -12,6 +12,7 @@ <h1>Batches</h1>
           <th>ID</th>
           <th>Name</th>
           <th>State</th>
+          <th></th>
```

Comment:
Intentional?

Reply:
Yes. I thought we didn't need a header that said cancel when the buttons say cancel.

Reply:
Oh yeah, of course.